templates/plugin/snmp_agent.conf.epp

```diff
@@ -0,0 +1,107 @@
+<%- | Optional $data = undef,
```

Comment:
please add a comment like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
Added

Reply:
This template is broken since your squash. I think the newline needs to go away?

Reply:
Your right sorry for that. Pushed a fix but the build failed on a totally different issue. It doesn't look like its caused by anything in the PR.